content/docs/user-guide/data-management/data-versioning.md

```diff
@@ -41,3 +41,33 @@ To simplify remote data operations, DVC now supports native versioning of files
 and directories on several cloud providers. This means that you can browse your
 files normally as you would see them in your local workspace.
 -->
+
+## Project configuration
+
+Besides metafiles, <abbr>DVC projects</abbr> may contain a config file
+(`.dvc/config`) that can also be treated as code when it comes to version
+control.
+
+<admon icon="book">
+
+See `dvc config` for more information on DVC config.
+
+</admon>
+
+Some times it's important to version configuration changes along with
+corresponding data updates. Most notably, if you [set up remote storage] and
+`dvc push` data for others to `dvc pull` later, you should `git commit` both the
+metafile(s) and `.dvc/config` to the repo.
+
+Advanced situations where this may also be necessary:
+
+- When migrating to a [shared cache]
+- If you enable `dvc cache hydra` and then create an [experiment queue] with
+  [param composition] to run from another environment
```

Comment:
```suggestion
- If you enable `dvc config hydra` and then create an [experiment queue] with
  [param composition] to run from another environment
```

I think we can drop this because:
1. There's no way to share a queue like this right now.
2. Once you queue the experiments, it doesn't matter whether you run them with hydra enabled (the hydra part is done).

What about `parsing` instead? If you have a `parsing` option set, it will impact how your `dvc.yaml` gets parsed and should be shared with everyone.

Reply:
Good idea, changed! But I thought `parsing` options only affect dictionary unpacking? (not how dvc.yaml gets parsed). Still a valid case here either way.

Reply:
Right, they affect dictionary unpacking. So you could have the same `dvc.yaml` generate two different commands depending on the value of the `parsing` options.

Reply:
Oh OK I guess that is why it's called "parsing". That work made me think more of reconstructing the DAG really. Anyway... This should be good now.